src/Orchard.Cms.Web/Modules/Orchard.OpenId/README.md

```diff
@@ -10,20 +10,30 @@ Flows supported: [code/implicit/hybrid flows](http://openid.net/specs/openid-con
 
 ## Configuration
 
-Configuration can be set through Open Id menu in the admin dashboard and also through a recipe step.
+Configuration can be set through OpenID Connect settings menu in the admin dashboard and also through a recipe step.
 
 Available settings are:
 + Testing Mode: Enabling Testing mode, removes the need of providing a certificate for signing tokens providing an ephemeral key. Also removes the requirement of using an HTTPS for issuing tokens.
 + Token Format: there are two options:
   + JWT: This format uses signed JWT standard tokens (not encrypted). It requires the SSL certificate being used is accepted as a trusted certificate by the client.
-  + Encrypted: This format uses non standard opaque tokens encrypted by the ASP.NET data protection block. 
+  + Encrypted: This format uses non standard opaque tokens encrypted by the ASP.NET data protection block. So, it doesn't require the client accept the SSL certificate as a trusted certificate.
 + Authority: Orchard url used by orchard to act as an identity server.
 + Audiences: Urls of the resource servers for which the identity server issues valid JWT tokens.
 + Certificate Store Location: CurrentUser/LocalMachine https://msdn.microsoft.com/en-us/library/system.security.cryptography.x509certificates.storelocation(v=vs.110).aspx
 + Certificate Store Name: AddressBook/AuthRootCertificateAuthority/Disallowed/My/Root/TrustedPeople/TrustedPublisher https://msdn.microsoft.com/en-us/library/system.security.cryptography.x509certificates.storename(v=vs.110).aspx
 + Certificate Thumbprint: The thumbprint of the certificate (It is recommended to not use same certificate it is been used for SSL).
++ Enable Token Endpoint: Enables action /Orchard.OpenId/Access/Token.
```

Comment:
I don't think mentioning the actual urls is interesting in this context. I would only show them if they are actually necessary to know to configure external services, and in this case I would create a custom doc section. And if this url should be well-known, a more user friendly one would make more sense.